application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -67,7 +67,7 @@ public static void disableButtons(@NotNull Message message) {
      * Escapes every markdown content in the given string.
      *
      * If the escaped message is sent to Discord, it will display the original message.
-     * 
+     *
```

Comment:
Add the space back

Reply:
Lmao. Undoes a space issue at `ActionRecord` and introduces a new space issue at a totally new place 😄

Reply:
Space was removed here too when it was gone in ActionRecord \:p